src/server/trpc/router/recipes.ts

```diff
@@ -34,7 +34,16 @@ export const recipesRouter = router({
   getRecipe: protectedProcedure
     .input(getRecipeSchema)
     .query(async ({ ctx, input }) => {
-      return await getRecipe(ctx, input.recipeId);
+      const recipe = await getRecipe(ctx, input.recipeId);
+      const roundedDate = getFormattedUtcDate();
+      const signedUrl = await getImageSignedUrl(
+        recipe!.authorId,
```

Comment:
Instead of the ```!``` operator, you can check if (!recipe) throw. There's an example for addRecipe below.

Reply:
Fixed.